static/js/ducks/spectra.js

```diff
@@ -6,18 +6,40 @@ import store from "../store";
 export const FETCH_SOURCE_SPECTRA = "skyportal/FETCH_SOURCE_SPECTRA";
 export const FETCH_SOURCE_SPECTRA_OK = "skyportal/FETCH_SOURCE_SPECTRA_OK";
 
+export const UPLOAD_SPECTRUM = "skyportal/UPLOAD_SPECTRUM";
+export const UPLOAD_SPECTRUM_OK = "skyportal/UPLOAD_SPECTRUM_OK";
+
+export const PARSE_SOURCE_SPECTRUM_ASCII =
+  "skyportal/PARSE_SOURCE_SPECTRUM_ASCII";
+export const PARSE_SOURCE_SPECTRUM_ASCII_OK =
+  "skyportal/PARSE_SOURCE_SPECTRUM_ASCII_OK";
+
+export const RESET_PARSED_SPECTRUM = "skyportal/RESET_PARSED_SPECTRUM";
+
 export function fetchSourceSpectra(id) {
   return API.GET(`/api/sources/${id}/spectra`, FETCH_SOURCE_SPECTRA);
 }
 
+export function parseASCIISpectrum(data) {
+  return API.POST(
+    `/api/spectrum/parse/ascii`,
+    PARSE_SOURCE_SPECTRUM_ASCII,
+    data
+  );
+}
+
+export function uploadSpectrumAscii(data) {
```

Comment:
Maybe pick a capitalization pattern and use it in both of these action creator names? (Ascii and ASCII)

Reply:
done